packages/schematics/angular/workspace/files/tsconfig.json.template

```diff
@@ -3,11 +3,11 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",<% if (strict) { %>
-    "noImplicitAny": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
     "noImplicitReturns": true,
-    "noImplicitThis": true,
     "noFallthroughCasesInSwitch": true,
-    "strictNullChecks": true,<% } %>
+    "strictPropertyInitialization": false,<% } %>
```

Comment:
why? Is this because of @Components with @Inputs?

I do think that developer's should provide a default value. That's what I do and it works quite well.

I recommend removing this line.

Reply:
Yeah that’s the reason. 

I’ll also do a PR In AIO to remove that.

Reply:
So for AIO, we better do a followup task, as there a quite a lot of errors when I removed `"strictPropertyInitialization": false`